main.go

```diff
@@ -86,12 +86,12 @@ func main() {
 
 		for _, account := range accountCRList {
 			logger := log.WithValues("AccountName", account.Name, "AccountID", account.Spec.AwsAccountID)
-
+			logger.Info("New Account being shredder") // Usefull for keeping track of when work begins on an account
```

Comment:
Shredded

Reply:
Probably should read something like "Account $ACCOUNT_ID from $ACCOUNT_CR being shredded"

Reply:
Thanks for pointing out the typo. 
I am sorry but it makes no sense to add account_id and account_cr in this log since those two pieces of information are already being logged `logger := log.WithValues("AccountName", account.Name, "AccountID", account.Spec.AwsAccountID)`

Reply:
I thought you wanted this so you could easily see the beginning of an account shred but if you already have the account id and cr name than this is useless

Reply:
Say I'm grepping through logs. 
`oc logs podname | grep "being shredded" | grep account_cr_name`
vs
`oc logs podname| grep "account_cr_name being shredded"`

Simplifies searching to a single string than having to grep for multiple strings. The same thought can be applied to a jq select.

Reply:
`oc logs podname | grep account_cr_name`

Reply:
But that doesn't automatically show you the top of a specific run. An account CR will be run multiple times in the operator logs